### DISCUSSION TITLE
Will `createMutable` be remove from solidjs in the future?

### DISCUSSION THREAD

**Discussion by xieyuheng** [solidjs/solid#2288] [Q&A] [answered]
Or is it guaranteed that `createMutable` will stay in all future version of solidjs?

I ask because:

- I heard of removing it in chat years ago.
- I see it is not even mentioned in the Learn part of the new docs.

**Comment by atk** [comment, marked as answer]
We prefer stores over mutables for the reasons laid out in the [information box about the method](https://docs.solidjs.com/reference/store-utilities/create-mutable), but it will not be removed without being deprecated for some time before. And even then it will probably just be moved into a [community primitive](https://primitives.solidjs.community/).

**Reply by xieyuheng** [reply]
Thanks for the reply.
